### PR TITLE
Feat/create static UI

### PR DESCRIPTION
**Pull request by minhng271**
Create static UI for:
- Title
- Form input task
- Button toggle-all
- Todo-list
- Todo-count
- Button All
- Button Active
- Button Completed

### CODE REVIEW

File: javascript/big-practice/src/index.html [hunk 1]
```diff
@@ -5,10 +5,34 @@
...
+      <label for="toggle-all"></label>
+      <ul class="todo-list"></ul>
+      <footer class="footer">
+        <span class="todo-count"></span>
+        <ul class="filters">
+          <li>
+            <a href="" class="selected">All</a>
+          </li>
+          <li>
+          <a href="">Active</a>
```
Comment (thanhnguyen-agilityio) [inline javascript/big-practice/src/index.html:27]: wrong code format

File: javascript/big-practice/src/index.html [hunk 2]
```diff
@@ -5,10 +5,34 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="styles/main.css">
-  <title>todomvc</title>
+  <title>Todo MVC</title>
 </head>
 <body>
-  <h1>abc</h1>
-  <script src="scripts/main.js"></script>
+  <div class="todoapp">
```
Comment (thanhnguyen-agilityio) [inline javascript/big-practice/src/index.html:11]: div contain a header is not suitable, should be section

File: javascript/big-practice/src/styles/abstracts/variables.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+:root {
+  /* Color */
+  --background: #f5f5f5;
+  --text: #4d4d4d;
+  --title:  rgba(175, 47, 47, 0.15);
+  --button: #777;
```
Comment (thanhnguyen-agilityio) [inline javascript/big-practice/src/styles/abstracts/variables.css:6]: need more specific, what attr for button? color, background, outline?

File: javascript/big-practice/src/styles/abstracts/variables.css [hunk 4]
```diff
@@ -0,0 +1,14 @@
+:root {
+  /* Color */
+  --background: #f5f5f5;
+  --text: #4d4d4d;
```
Comment (thanhnguyen-agilityio) [inline javascript/big-practice/src/styles/abstracts/variables.css:4]: more specific, the variable `text, title` are too general.

File: javascript/big-practice/src/styles/components/input.css [hunk 5]
```diff
@@ -0,0 +1,6 @@
+.todoapp input::-webkit-input-placeholder {
+  /* Chrome/Opera/Safari */
```
Comment (thanhnguyen-agilityio) [inline javascript/big-practice/src/styles/components/input.css:2]: I am not sure this project will support all of those browsers

File: javascript/big-practice/src/index.html [hunk 6]
```diff
@@ -1,14 +1,47 @@
...
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="styles/main.css">
-  <title>todomvc</title>
-</head>
-<body>
-  <h1>abc</h1>
-  <script src="scripts/main.js"></script>
```
Comment (meovan07) [inline javascript/big-practice/src/index.html:12]: I don't see your script in index.html anymore please check

File: javascript/big-practice/src/styles/abstracts/variables.css [hunk 7]
```diff
@@ -0,0 +1,25 @@
+:root {
+  /* Color */
+  --primary-light: rgb(245, 245, 245);
+  --primary-dark: rgb(246, 246, 246);
+  --normal: #4d4d4d;
```
Comment (hamyy2912) [inline javascript/big-practice/src/styles/abstracts/variables.css:5]: for colors, should only use 1 type of value (color name or rgb or hex,...)

File: javascript/big-practice/src/styles/components/input.css [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+.new-todo {
+  padding: 16px 16px 16px 60px;
+  border: none;
+  background: var(--dark-003);
+  box-shadow: inset 0 -2px 1px var(--dark-03);
+  font-size: 24px;
+  font-family: inherit;
+  font-weight: inherit;
+  line-height: 1.4em;
+  border: 0;
```
Comment (hamyy2912) [inline javascript/big-practice/src/styles/components/input.css:16]: duplicated border property in line 9 and 16

File: javascript/big-practice/src/styles/components/toggle.css [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+  top: 15px;
+  left: -13px;
+  transform: rotate(90deg);
+}
+
+.toggle-all + label:before {
+  content: "❯";
+  font-size: 22px;
+  color: var(--platinum);
+  padding: 10px 27px 10px 27px;
```
Comment (hamyy2912) [inline javascript/big-practice/src/styles/components/toggle.css:20]: if padding top and bottom are equal, right and left are equal, you can write `padding: 10px 27px` for shortcut
check another places too

**Review by hamyy2912 — APPROVED** [review]

**Pull request merged**